src/day8/re_frame/less/main.less

```diff
@@ -0,0 +1,108 @@
+#--re-frame-trace-- {
+  background: white;
+  color: black;
+  font-family: 'courier new', monospace;
+
+  tbody {
+    color: #aaa;
+  }
+  tr:hover {
+    transition: all 0.1s ease-out;
+    background: aliceblue;
+    filter: brightness(90%);
+  }
+  tr:nth-child(even) {
+    background: aliceblue;
+  }
+  .button {
+    padding: 5px 5px 3px;
+    margin: 5px;
+    border-radius: 2px;
+    cursor: pointer;
+  }
+  .text-button {
+    border-bottom: 1px dotted #888;
+    font-weight: normal;
+  }
+  .button:focus, .text-button:focus {
+    border-radius: 2px 2px 0 0;
+    -webkit-box-shadow: inset 0px -5px 0px 0px rgba(0,0,0,0.3);
+    -moz-box-shadow: inset 0px -5px 0px 0px rgba(0,0,0,0.3);
+    box-shadow: inset 0px -5px 0px 0px rgba(0,0,0,0.3);
+  }
+  .icon-button {
+    font-size: 10px;
+  }
+  button.tab {
+
+  }
+  .tab {
+    background: transparent;
+    border-radius: 0;
+    text-transform: uppercase;
+    font-family: monospace;
+    letter-spacing: 2px;
+    margin-bottom: 0;
+    padding-bottom: 4px;
+    vertical-align: bottom;
+  }
+  .tab.active {
+    background: transparent;
+    border-bottom: 3px solid lightblue;
+    border-radius: 0;
+    padding-bottom: 1px;
+  }
+  ul.filter-items {
+    list-style-type: none;
+    padding: 0;
+    margin: 0 -5px;
+    margin-top: 10px;
+  }
+  .filter-items li {
+    color: #333;
+    background: #efefef;
+    display: inline-block;
+    font-size: 0.9em;
+    margin:  5px;
+  }
+  .filter-items {
+    li {
+      .filter-item-string {
+        color: #616cdb;
```

Comment:
Since we are now using less we can put colors hexes in variables and give them descriptive names!

Reply:
Yes, true! CSS preprocessing is great!